adele-messaging/src/main/java/com/microservicesteam/adele/messaging/listeners/DeadEventListener.java

```diff
@@ -22,6 +26,7 @@ public DeadEventListener(EventBus eventBus) {
 
     @Subscribe
     public void handleEvent(DeadEvent event) {
+        LOGGER.debug("Dead event: {}", event);
```

Comment:
Btw, I would investigate whether google bus provide logging at event consumption. We might need to enable only in the config.